com.unity.multiplayer.mlapi/Tests/Runtime/RPCQueueTests.cs

```diff
@@ -0,0 +1,109 @@
+using System.Collections;
+using NUnit.Framework;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+using UnityEngine.TestTools;
+using MLAPI.SceneManagement;
+using MLAPI.Transports.UNET;
+
+namespace MLAPI.RuntimeTests
+{
+    public class RPCQueueTests
+    {
+        private NetworkingManager m_NetMan;
+
+        [UnitySetUp]
+        public IEnumerator SetUp()
+        {
+            yield return new EnterPlayMode();
+        }
+
+        /// <summary>
+        /// RPCQueueUnitTest
+        /// Tests the egress and ingress RPC queue functionality
+        /// ** This does not include any of the MLAPI to Transport code **
+        /// </summary>
+        /// <returns>IEnumerator</returns>
+        [UnityTest]
+        public IEnumerator RPCQueueUnitTest()
+        {
+            GameObject NetManObject = new GameObject();
```

Comment:
```suggestion
            var netManObject = new GameObject();
```

we might use `var` in these kinds of scenarios where we know the type for sure on the right side.